### PR TITLE
samqfs: update 5.64.5

### PR DESCRIPTION
**Pull request by cgrzemba**
- all userland services now build in 64bit
	- read LTO loadtimeout from drive to support LTO9
	- keep RPC API compatible with OHSM
	- use GCC10 for build
	- add version to kernel modules

### CODE REVIEW

File: components/openindiana/samqfs/Makefile [hunk 1]
```diff
@@ -35,9 +35,9 @@ TEST_TARGET = $(NO_TESTS)
 
 include $(WS_MAKE_RULES)/common.mk
 
-COMPONENT_BUILD_ARGS = DEBUG=no MYSQL_VERSION=mariadb/10.3 SAMQFS_VERSION=$(COMPONENT_VERSION)$(BRANCH_VERSION)
+COMPONENT_BUILD_ARGS = AMD64=yes DEBUG=no MYSQL_VERSION=mariadb/10.3 SAMQFS_VERSION=$(COMPONENT_VERSION)$(BRANCH_VERSION) NO_BUILD_ACSLS=-D_NoACSLS_
```
Comment (mtelka) [inline components/openindiana/samqfs/Makefile]: Could you please rebuild this with mariadb 10.6?  Thank you.

**Review by mtelka — CHANGES_REQUESTED** [review]

File: components/openindiana/samqfs/Makefile (hunk 1 above)
Comment (mtelka) [reply components/openindiana/samqfs/Makefile]: With a bit of luck `MYSQL_VERSION=$(MYSQL_IMPLEM)/$(MYSQL_VERSION)` should work, I think.

Comment (cgrzemba) [reply components/openindiana/samqfs/Makefile]: works, Thanks!

**Comment by cgrzemba** [comment]
works, Thanks!

**Pull request merged**